lms/static/js/jquery.gradebook.js

```diff
@@ -27,7 +27,10 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		// Updating direction of scroll by calculating pixels.If the value is 0,
+		// the gradebook should be moved all the way to the left side of the screen.
+		// maxScroll is difference between table width and its parent width.
```

Comment:
@singingwolfboy please see this https://github.com/edx/edx-platform/pull/10021#discussion_r41270362